docs: update playground docs (#4704)
langflow-ai/langflow: docs/docs/Workspace/workspace-playground.md

:::

## Memory Management {#821a5ee2a8a44d35b49943cba630511c}

---

> Click on any chat session menu on the right, then click on the message logs option to open a table of previous interactions for a specific flow.

Reviewer: To view messages by session ID, from the **Playground**, click the **Options** menu of any session, and then select **Message Logs**.

> Whenever you send a message from the **Playground** interface, in the **Message Logs** option you'll see a table of previous interactions for that session.

Reviewer: ```suggestion
When you send a message from the **Playground** interface, the interactions for that session are stored in the **Message Logs**.
```

}'
```

> Check your flow's playground. In addition to the messages stored for the Default Session, a new session is started with your new Session ID.

Reviewer: ```suggestion
Check your flow's **Playground**. In addition to the messages stored for the Default Session, a new session is started with your new Session ID.
```
